lib/ansible/module_utils/network/ios/ios.py

```diff
@@ -107,6 +107,12 @@ def get_defaults_flag(module):
         return ['full']
 
 
+def get_diff(module, candidate=None, running=None, match='line', diff_ignore_lines=None, path=None, replace='line'):
```

Comment:
Since the api in ios cliconf plugin set the default value https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/cliconf/ios.py#L55 not sure if this additional get_diff() function in module_utils is required?